### PR TITLE
matthew yang basics-10-1 SPS(basic) [PART 1 & 2]

### PR DESCRIPTION
**Pull request by datatango**
Please fill out the survey before submitting the pull request. Thanks!

🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀

How many hours did you spend on this assignment?
>>2 hours

Please fill in one error and/or error message you received while working on this assignment.
>> many undefined errors

What part of the assignment did you spend the most time on?
>> addressing undefined errors

Comfort Level (1-5): 3

Completeness Level (1-5): 3

What did you think of this deliverable?
>>Allows me to apply the concepts learnt during the course.

Is there anything in this code that you feel pleased about?
>> No comment

### CODE REVIEW

**Comment by datatango** [comment]
Please fill out the survey before submitting the pull request. Thanks!

🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀

How many hours did you spend on this assignment?

many many hours

Please fill in one error and/or error message you received while working on this assignment.

global variables not appearing in default stats message

What part of the assignment did you spend the most time on?

getting global variables to work

Comfort Level (1-5): 3

Completeness Level (1-5): 5

What did you think of this deliverable?

it tested me

Is there anything in this code that you feel pleased about?

no

File: script.js [hunk 1]
```diff
@@ -1,4 +1,115 @@
...
+var numPlayerWinCounter = 0;
+var numComputerWinCounter = 0;
+var numDrawsCounter = 0;
+
+// game modes
+
+var gameMode1 = `user to log in`;
+var gameMode2 = `user to throw`;
+var gameMode3 = `user to play`;
+gameMode = gameMode1;
```
Comment (eddiejpot) [inline script.js:12]: Goo work declaring the gameModes at the top!

File: script.js [hunk 2]
```diff
@@ -1,4 +1,115 @@
...
+  if (computerPick == 2) {
+    computerAction = `paper`;
+  }
+
+  if (computerPick == 3) {
+    computerAction = `stone`;
+  }
+
+  return computerAction;
+};
```
Comment (eddiejpot) [inline script.js:38]: Goo work with this function! Would be better to name this function more accurately. 

e.g. `generateSPS `

File: script.js [hunk 3]
```diff
@@ -1,4 +1,115 @@
...
+    (gameMode == gameMode3 &&
+      userThrow == `scissors` &&
+      computerResult == `paper`) ||
+    (gameMode == gameMode3 &&
+      userThrow == `paper` &&
+      computerResult == `stone`) ||
+    (gameMode == gameMode3 &&
+      userThrow == `stone` &&
+      computerResult == `scissors`)
+  ) {
```
Comment (eddiejpot) [inline script.js:94]: Nice work on using conditionals!

**Review by eddiejpot — COMMENTED** [review]
Over all good job! Looking forward to project 2 which will involve a different data structure !

**Pull request closed without merging**